scripts/install-cni.sh

```diff
@@ -192,7 +192,7 @@ else
 fi
 
 # Set mtu
-default_nic=$(ip route get 8.8.8.8 | sed -nr "s/.*dev ([^\ ]+).*/\1/p")
+default_nic=$(route -n | grep -E '^0\.0\.0\.0\s+\S+\s+0\.0\.0\.0' | grep -oE '\S+$')
```

Comment:
For easier future reference, could you put a quick comment on what this is matching? This was not immediately clear to me until I tried out the command and learned the output format.